app/design/frontend/Candleaf/main_theme/Magento_Theme/web/css/source/_module_extend.less

```diff
@@ -0,0 +1,372 @@
+/***************
+Hero
+****************/
+& when (@media-common = true) {
+    .hero {
+        &-content {
+            color: @color-gray-dark1;
+        }
+    }
+}
+
+//Mobile
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m){
+    .hero {
+        background-image: url("../images/hero-bg-image-mobile.jpg");
+        background-color: #cccccc;
+        height: 780px;
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: cover;
```

Comment:
background: #cccccc url("../images/hero-bg-image-mobile.jpg") center/cover no-repeat;

Reply:
for #cccccc use variable